superset-frontend/stylesheets/superset.less

```diff
@@ -582,3 +582,35 @@ hr {
   background-image: url('../images/icons/error_solid_small_red.svg') !important;
   background-position: -2px center !important;
 }
+
+// Ant-d overrides since these are injected as inline styles and can't
+// be overrided in emotion
+.ant-menu-submenu.ant-menu-submenu-popup.ant-menu.ant-menu-light.ant-menu-submenu-placement-bottomLeft {
+  top: 51px !important;
+  margin-left: -2px !important;
+  @media (max-width: 767px) {
+    top: 269px !important;
+  }
+  & > .ant-menu {
+    border-radius: 0px !important;
+  }
+}
+
+.ant-menu-submenu.ant-menu-submenu-popup.ant-menu.ant-menu-light {
+  top: 51px !important;
+  border-radius: 0px !important;
+  @media (max-width: 767px) {
+    top: 269px !important;
+  }
+}
+
+.about-section .version-info {
```

Comment:
I don't believe we need to add these styles to `superset.less` file

In this PR, they're still part of `Menu.tsx` on lines [92-102](https://github.com/apache/superset/pull/14184/files#diff-e7a361a8592b5429f76b1001f1c4d1e7368c3af223564624671b986c0eb02215R92-R102)

I _suspect_ those are not being applied because of the file they're in... but if you were to add them in `MenuRight.tsx` they'll probably work (with the theme vars, too).